clifford/tools/__init__.py

```diff
@@ -130,12 +133,17 @@ def mat2Frame(A: np.ndarray,
     A : ndarray
         MxN matrix representing vectors
 
+    I: None, pseudoscalar of the frame
```

Comment:
The first line should be the type, not a description. Something like:
```suggestion
    I : MultiVector
```